modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/search/SearchTransactionRequest.scala

```diff
@@ -0,0 +1,19 @@
+package org.tessellation.rosetta.server.model.search
+
+import org.tessellation.schema.address.Address
+import org.tessellation.security.hash.Hash
+
+import eu.timepit.refined.types.numeric.{NonNegLong, PosLong}
+
+/** This is an intermediary object used to hold the values from the Rosetta request that is relevant to our system.
+  */
+
+case class SearchTransactionRequest(
+  operation: Operator,
+  address: Option[Address],
+  networkStatus: Option[OperationStatus],
```

Comment:
isn't `network status` and unchanged name?

Reply:
`networkStatus` is the same as `OperationStatus`. The `OperationStatus` class has supplanted the now deleted `NetworkStatus` class

Reply:
exactly, and you left the network status name as the class' field name. IMO it should be either just status or operation status, not network status.

Reply:
I'll change it to `status`.